code/UI/Communication/QuickChat/QuickChat.cs

```diff
@@ -116,7 +116,7 @@ private void BuildInput()
 			return;
 
 		var keyboardIndexPressed = InventorySelection.GetKeyboardNumberPressed();
-		if ( keyboardIndexPressed <= 0 ) // Only accept keyboard numbers 1-9
+		if ( keyboardIndexPressed <= 0 ) // moved the comment that was here into doc
```

Comment:
```suggestion
		if ( keyboardIndexPressed <= 0 )
```
We won't need a comment here at all anymore.